sbt

```diff
@@ -171,6 +171,7 @@ acquire_sbtn () {
   local target="$p/sbtn"
   local archive_target=
   local url=
+  local arch = "x86_64"
```

Comment:
@eed3si9n `arch` needs to be defined as default, otherwise, if not using linux, it will be undefined in below in line 199 below (`echoerr "downloading sbtn ${sbtn_v} for ${arch}"`)

Reply:
this doesn't work on mac, it should be `local arch="x86_64"`:

> /opt/homebrew/Cellar/sbt/1.8.2/libexec/bin/sbt: line 174: local: `=': not a valid identifier